package.json

```
@@ -10,7 +10,7 @@
     "url": "https://github.com/nodejs/corepack.git"
   },
   "engines": {
-    "node": "^18.17.1 || >=20.10.0"
+    "node": "^18.17.1 || ^20.10.0 || >=22.0.0"
```

nit: we could restrict it to the first LTS version of 22.x
```suggestion
    "node": "^18.17.1 || ^20.10.0 || >=22.10.0"
```